spec-tests/test_meta.rs

```diff
@@ -83,7 +83,7 @@ pub enum Runner {
 
 impl Runner {
     pub fn should_ignore(&self) -> bool {
-        matches!(self, Self::ForkChoice | Self::Kzg | Self::Sync)
+        matches!(self, Self::ForkChoice | Self::LightClient | Self::MerkleProof | Self::Sync)
```

Comment:
I updated these on `main`, so if you rebase, there should just be the deletion for `Kzg`